FastSurferCNN/data_loader/conform.py

```diff
@@ -85,99 +113,119 @@ def make_parser() -> argparse.ArgumentParser:
-    parser.add_argument("--order", dest="order", help=h_order, type=int, default=1)
     parser.add_argument(
         "--check_only",
         dest="check_only",
         default=False,
         action="store_true",
-        help="If True, only checks if the input image is conformed, and does not "
-             "return an output.",
+        help="Specifies that to only check whether the input image is conformed, and do not write an output image.",
     )
     parser.add_argument(
         "--seg_input",
         dest="seg_input",
-        default=False,
         action="store_true",
-        help="Specifies whether the input is a seg image. If true, the check for "
-             "conformance disregards the uint8 dtype criteria. Use --dtype any for "
-             "equivalent results. --seg_input overwrites --dtype arguments.",
+        help="Specifies that the input is a seg image: The *default values* for dtype and rescale are changed to "
+             "'integer' and 'none', which only means the dtype must be an integer and no rescaling is performed.",
     )
     parser.add_argument(
         "--vox_size",
         dest="vox_size",
+        metavar="<float>|min|any",
         default=1.0,
         type=__vox_size,
-        help="Specifies the target voxel size to conform to. Also allows 'min' for "
-             "conforming to the minimum voxel size, otherwise similar to mri_convert's "
-             "--conform_size <size> (default: 1, conform to 1mm).",
+        help="Specifies the target voxel size to conform to (default: 1, conform to 1mm). Options: <float> between 0 "
+             "and 1 (target voxel size, isotropic, similar to mri_convert's --conform_size <size>); 'min' (conform to "
+             "the minimum voxel size); 'any' (ignore this criteria, accept any voxel size even non-isotropic).",
     )
     parser.add_argument(
         "--conform_min",
-        dest="conform_min",
-        default=False,
-        action="store_true",
-        help="Specifies whether the input is or should be conformed to the "
-        "minimal voxel size (used for high-res processing) - overwrites --vox_size.",
+        dest="vox_size",
+        action="store_const",
+        const="min",
+        help="(Legacy, prefer --vox_size min for same functionality) Specifies that the image should be conformed to "
+             "the minimal voxel size (used for high-res processing) -- overwrites --vox_size.",
+    )
+    parser.add_argument(
+        "--img_size",
+        dest="img_size",
+        default="auto",
+        metavar="<int>|auto|fov|any",
+        type=__img_size,
+        help="Specifies the image size to conform to. Options: <int> (dimension of the target image), 'auto' "
+             "(determine dimensions of image from largest field-of-view dimension, min. 256), 'fov' (determine "
+             "dims. of image from field of view, dims. may be differ) or 'any' (ignore this criteria, similar to fov).",
```

Comment:
not toally clear here what the difference between auto and fov is. maybe auto is always a cube ?

Reply:
"auto" is basically fov but expanded to a cube, "fov" keeps the original field of view